src/main/resources/com/welovecoding/nbeditorconfig/FontAndColors.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE fontscolors PUBLIC "-//NetBeans//DTD Editor Fonts and Colors settings 1.1//EN" "http://www.netbeans.org/dtds/EditorFontsColors-1_1.dtd">
+<fontscolors>
+  <fontcolor name="comment" default="comment"/>
+  <fontcolor name="keyword" default="keyword"/>
+  <fontcolor name="whitespace" default="whitespace"/>
+  <fontcolor foreColor="ff509a05" name="section"/>
+  <fontcolor name="section_delimiter" default="section">
+    <font style="bold"/>
+  </fontcolor>
+  <fontcolor foreColor="ff000000" name="equals"/>
+</fontscolors>
```

Comment:
We really need an `.editorconfig` in our project which defined final new lines for xml files. :grinning: